test/metabase/api/embed_test.clj

```diff
@@ -155,7 +155,7 @@
 
 (def successful-dashboard-info
   {:auto_apply_filters true, :description nil, :parameters [], :dashcards [], :tabs [],
-   :param_values {}, :param_fields nil})
+   :param_values {}, :param_fields nil :width "fixed"})
```

Comment:
Adding the :width "fixed" default here should allow the tests to pass. Since we now send that key via the api, it's fair to add it as part of the expected successful dashboard info.